itou/prescribers/tests.py

```diff
@@ -644,3 +648,44 @@ def test_update_kind(self):
             ).count()
         )
         assert 1 == PrescriberOrganization.objects.filter(kind=PrescriberOrganizationKind.OTHER).count()
+
+
+@pytest.mark.parametrize("organization_kind", PrescriberOrganizationKind.values)
```

Comment:
_nit_ : Théoriquement le `.values` est de trop puisque les clés sont les membres de l'enum et non leur valeur, mais aussi parce que l'itération sur un enum retourne les membres de celui-ci donc ça ne devrais rien changer pour les `TextChoice`.
```suggestion
@pytest.mark.parametrize("organization_kind", PrescriberOrganizationKind)
```